### ISSUE TITLE
[Bug] chart type 'graph' drag area is too limited

### ISSUE THREAD

**Issue by skayi** [apache/echarts#17214]
### Version

4.8.0

### Link to Minimal Reproduction

https://echarts.apache.org/examples/zh/editor.html?c=graph-simple

### Steps to Reproduce

<img width="1130" alt="image" src="https://user-images.githubusercontent.com/61896686/173507654-ca34240b-b32c-49cd-ab65-c3e4475065ca.png">

**Cannot drag chart outside red box**

### Current Behavior

can drag chart only inside red box

### Expected Behavior

can drag chart inside & outside red box

### Environment

```markdown
- OS:
- Browser:
- Framework:
```

**Comment by Ovilia** [comment]
This is by-design but I do have similar feelings that we should provide a way to enable the roaming outside of component bounding box if it shall not be the default behavior. Similar for map.

I remember having this discussion with @100pah and @pissang who may have different opinion. Maybe it's time to start new discussion on this.

**Comment by warflash** [comment]
Is there a discussion/status we can follow for updates on this? The current behaviour feels really unintuitive to me so trying to find a solution 🙏

**Comment by warflash** [comment]
This should not be closed imo, atleast not without linking to the mentioned possible new discussion

**Comment by giacomotontini** [comment]
Agree. When dragging the map the restricted bounding box is counterintuitive

**Comment by elya5** [comment]
<img width="720" height="542" alt="Image" src="https://github.com/user-attachments/assets/1f578ffd-2de1-407f-b5ad-d02d5473bff8" />

In this bigger graph, more or less only the area marked in red is draggable. The rest is not. I think the draggable area should at least include all nodes.

**Comment by elya5** [comment]
Ah, `roamTrigger: 'global'` fixed that for me.
